packages/rest-typings/src/v1/integrations/IntegrationsCreateProps.ts

```diff
@@ -70,7 +70,7 @@ const integrationsCreateSchema = {
 				},
 				overrideDestinationChannelEnabled: {
 					type: 'boolean',
-					nullable: false,
+					nullable: true,
```

Comment:
if ajv is being used, then why Match too?

Reply:
~i was afraid of removing it and creating a new issue~ done!